src/main/java/org/broadinstitute/gatk/nativebindings/NativeLibrary.java

```diff
@@ -0,0 +1,21 @@
+package org.broadinstitute.gatk.nativebindings;
```

Comment:
I think `org.broadinstitute.gatk.nativebindings` is fine given that the repo is "gatk-native-bindings" (ie., native bindings for the GATK specifically), and given that the hellbender package will eventually be renamed to gatk.